lib/imapSimple.js

```diff
@@ -92,6 +92,40 @@ ImapSimple.prototype.openBox = function (boxName, callback) {
     });
 };
 
+/**
+ * Close a mailbox
+ *
+ * @param {boolean} autoExpunge If autoExpunge is true, any messages marked as Deleted in the currently open mailbox will be remove
```

Comment:
this can be documented as optional by putting brackets around the variable, like

```js
@param {boolean} [autoExpunge=true]
```